sdk/go/common/workspace/plugins_test.go

```diff
@@ -379,18 +395,19 @@ func TestPluginDownload(t *testing.T) {
 	})
 }
 
+//nolint:paralleltest // mutates environment variables
 func TestPluginGetLatestVersion(t *testing.T) {
 	t.Run("Test GetLatestVersion From Pulumi GitHub Releases", func(t *testing.T) {
 		info := PluginInfo{
 			PluginDownloadURL: "",
-			Name:              "aws",
+			Name:              "mock-latest",
 			Kind:              PluginKind("resource"),
 		}
 		expectedVersion := semver.MustParse("4.37.5")
 		source := info.GetSource()
 		getHTTPResponse := func(req *http.Request) (io.ReadCloser, int64, error) {
 			assert.Equal(t,
-				"https://api.github.com/repos/pulumi/pulumi-aws/releases/latest",
+				"https://api.github.com/repos/pulumi/pulumi-mock-latest/releases/latest",
```

Comment:
https://github.com/pulumi/pulumi-mock-latest does not exist but that's OK i guess?

Reply:
I'm scratching my head here as to why I did this, but I think it was to make it clear it was a unit test with a fake URL.

Reply:
That's legit.